src/picongpu/submit/taurus-tud/k20x_profile.tpl

```diff
@@ -72,11 +72,6 @@ umask 0027
 mkdir simOutput 2> /dev/null
 cd simOutput
 
-# we are not sure if the current bullxmpi/1.2.4.3 catches pinned memory correctly
-#   support ticket [Ticket:2014052241001186] srun: mpi mca flags
-#   see bug https://github.com/ComputationalRadiationPhysics/picongpu/pull/438
-export OMPI_MCA_mpi_leave_pinned=0
```

Comment:
I had an offline discussion with @psychocoderHPC and I will check it with `ompi_info --all`.
Results will be posted soon.